### PR TITLE
Tutorial: added short example

### PR DESCRIPTION
**Pull request by anki-code**
Hi! xonsh is the best of the best of the best! Thanks!

I read articles about xonsh and almost every article has comments from people about xonsh subproc determination - i.e. how fun that `ls -l` could becomes `ls - l` Python-mode variables.

I added the short example at the "determination" section of the tutorial to give the information about strict syntax.

### CODE REVIEW

**Comment by anki-code** [comment]
@gforsyth @laloch @melund please help Anthony and merge this simple PR without his attention.

**Comment by anki-code** [comment]
@scopatz if you prepare release and all core is sleeping please merge this ))

File: docs/tutorial.rst [hunk 1]
```diff
@@ -315,10 +315,16 @@ The determination between the two modes is done well ahead of any execution.
 You do not need to worry about partially executed commands - that is
 impossible.
 
-If you absolutely want to run a subprocess command, you can always
-force xonsh to do so with the syntax that we will see in the following
-sections.
+If you still have concerns about the determination process and want to have strong confidence
+when running a subprocess command in your xonsh scripts, you can always use the elegant subprocess
+mode syntax that we will see in the following sections. Short example now:
```
Comment (scopatz) [inline docs/tutorial.rst:320]: The english here should be cleaned up a little

Comment (anki-code) [reply docs/tutorial.rst:320]: www.grammarly.com told me that
* Grammar - Correct
* Punctuation - Correct
No thoughts what you mean :)
Please fix, you have edit rights.

Comment (scopatz) [reply docs/tutorial.rst:320]: Yeah, the syntax is all correct. It is that the word choice and idioms feel a little off to me. Sorry, but I can barely keep up with all of the other issues.  I have to let other people help.

Comment (anki-code) [reply docs/tutorial.rst:320]: @gforsyth Gil, please help with idioms ;)

**Comment by anki-code** [comment]
Fresh example where @scopatz explains that there is a way to set strict syntax - https://news.ycombinator.com/item?id=24744653

File: docs/tutorial.rst (hunk 1 above)
Comment (scopatz) [reply docs/tutorial.rst:320]: How about something like:

```
If you would like to explicitly run a subprocess command, you can always
use the formal xonsh subprocess syntax that we will see in the following sections. For example:
```

Comment (gforsyth) [inline docs/tutorial.rst:320]: Maybe
```suggestion
If you know that you want to run a command in subprocess mode, you can force subprocess execution by using the following syntax:
```

Comment (anki-code) [reply docs/tutorial.rst]: Thanks! I've chosen the scopatz version. It has more strict text.

Comment (anki-code) [reply docs/tutorial.rst]: done

**Comment by scopatz** [comment]
Thanks @anki-code!

**Pull request merged**